docs/websites/cms/drupal/how-to-update-drupal-8-on-ubuntu-18-04/index.md

```diff
@@ -0,0 +1,137 @@
+
+1. Create a `backups` directory in your site's document root to store any backups you make of your Drupal site.
+
+        sudo mkdir /var/www/html/example.com/backups
+
+1. Create an archive of your existing site files and move it into the `backups` directory. Ensure you replace `example.com` with your own site's domain name.
+
+        cd /var/www/html/example.com/public_html
+        sudo tar -cvzf example.com-BCKP-$(date +%Y%m%d).tar.gz ./
+        sudo mv -v example.com-BCKP-*.tar.gz ../backups
+
+    {{< note >}}
+This process can also be scripted and run on a regular basis using [cron](/docs/tools-reference/tools/schedule-tasks-with-cron/).
+    {{</ note >}}
+
+## Download Updates
+
+You are now ready to check your Drupal system for available updates. Once you have identified the necessary updates, you will download them as an archive to your Linode.
+
+1.  Log in to your Drupal site and navigate to the [Admin Toolbar](https://www.drupal.org/project/admin_toolbar). Click on **Reports** and then on **Available updates**.
+
+    {{< note >}}
+If **Available updates** is not listed, enable the [Update Manager](https://www.drupal.org/docs/8/core/modules/update-manager) module by navigating to the **Extend** menu item in the Admin Toolbar. See [Drupal's documentation](https://www.drupal.org/docs/8/extending-drupal-8/installing-drupal-8-modules#s-step-2-enable-the-module) for more details on enabling modules.
+{{< /note >}}
+
+1.  Right click the link under the **RECOMMENDED VERSION** heading and copy the link address and paste it somewhere you can access later.
+
+    ![A Drupal Update](drupal-updates-download.png)
+
+    {{< note >}}
+If you receive an error when your Drupal 8 installation checks for available updates, it may be having issues communicating with the Drupal website to see their are updates. You can check your site's recent log messages, by navigating to **Reports** and selecting **Recent log messages** to further investigate the issue.
+    {{</ note >}}
+
+1.  Connect to your Linode over SSH:
+
+        ssh user@192.0.2.0
+
+1.  Navigate to your site's directory. Download the Drupal core archive using `wget` and pasting the link address you copied from Step 2. Ensure you replace `example.com` with your own site's directory name.
+
+        cd /var/www/html/example.com
+        sudo wget https://ftp.drupal.org/files/projects/drupal-8.8.2.tar.gz
+
+## Upgrade Your Site
+
+Now that the Drupal core archive is saved to your Linode, you are ready to proceed with the version upgrade of your site.
+
+###  Enable Maintenance Mode
+
+Drupal's *maintenance mode* allows users with the right permissions to use your site while all anyone else will be presented with a message that the site is under maintenance.
+
+1.  While logged into your Drupal site, navigate to **Configuration**. Under the **Development** heading, click on **Maintenance mode**.
+
+    ![Maintenance Mode](drupal-updates-maintenance.png)
```

Comment:
Outline Maintenance Mode or use an arrow to call it out.

Reply:
fised